e2e-tests/src/test/utility.rs

```diff
@@ -11,25 +8,15 @@ pub fn batch_transactions(config: &Config) -> anyhow::Result<()> {
 
     let (connection, to) = setup_for_transfer(config);
 
-    let call = compose_call!(
```

Comment:
We can use `transfer` https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/transfer.rs#L12 here